bugbug/models/invalid.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+
+from bugbug import feature_cleanup, issue_features, utils
+from bugbug.model import IssueModel
+
+logger = logging.getLogger(__name__)
+
+
+class InvalidModel(IssueModel):
+    def __init__(self, lemmatization=False):
+        super().__init__(
+            owner="webcompat", repo="web-bugs", lemmatization=lemmatization
+        )
+
+        self.calculate_importance = False
+
+        feature_extractors = []
+
+        cleanup_functions = [feature_cleanup.CleanCompatibilityReportDescription()]
+
+        self.extraction_pipeline = Pipeline(
+            [
+                (
+                    "report_extractor",
+                    issue_features.IssueExtractor(
+                        feature_extractors, cleanup_functions, rollback=False
+                    ),
+                ),
+                (
+                    "union",
+                    ColumnTransformer(
+                        [
+                            (
+                                "first_comment",
+                                self.text_vectorizer(min_df=0.0001),
+                                "first_comment",
+                            ),
+                        ]
+                    ),
+                ),
+            ]
+        )
+
+        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+
+    def get_labels(self):
+        classes = {}
+        for issue in self.github.get_issues():
```

Comment:
I think at some point we need to include data from the new reporter tool as well.

Reply:
Yeah, I plan on adding an ability to add and edit labels as soon as we more or less define triage process.
I believe I'll need to add a script to retrieve data from BigQuery as another source.